packages/mds-compliance-service/service/provider.ts

```diff
@@ -70,13 +73,16 @@ export const ComplianceServiceProvider: ServiceProvider<ComplianceService> & Pro
   createComplianceSnapshots: async complianceSnapshots => {
     try {
       const snapshots = await ComplianceRepository.createComplianceSnapshots(
-        complianceSnapshots.map(validateComplianceSnapshotDomainModel)
+        complianceSnapshots.map(validateComplianceSnapshotDomainModel),
+        async snapshots => {
+          // send to Kafka
+          const kafkaSnapshots = snapshots.map(snapshot => {
+            const { vehicles_found, ...kafkaSnapshot } = snapshot
+            return kafkaSnapshot
+          })
+          await ComplianceSnapshotStreamKafka.write(kafkaSnapshots)
```

Comment:
Oh wait, I take my approval back. Don't you need to create a `ComplianceSnapshotKafkaFailureEntity` if this fails?